src/index.js

```diff
@@ -130,20 +127,22 @@ export function app(state, actions, view, container) {
   function setElementProp(element, name, value, oldValue) {
     if (name === "key") {
     } else if (name === "style") {
-      for (var i in copy(oldValue, (value = value || {}))) {
-        element.style[i] = null == value[i] ? "" : value[i]
+      for (var i in copy(oldValue, value)) {
```

Comment:
Do we default to `{}`?

Seems we do?

Reply:
@selfup `copy` will return at least {}, the trick is in the assignment in the line below:

```js
element[name][i] = null == value || null == value[i] ? "" : value[i]
```

...here we make sure to access `value[i]` only if `value` exists. I found this way reduced a few bytes and kept the same functionality (tests passed).

Reply:
Ok awesome! Just making sure 🎉